guide/src/format/configuration/general.md

```diff
@@ -46,15 +46,19 @@ This is general information about your book.
   `src` directly under the root folder. But this is configurable with the `src`
   key in the configuration file.
 - **language:** The main language of the book, which is used as a language attribute `<html lang="en">` for example.
+- **logo:** Path to a logo to shown at the top of the navigation bar. If this
+  is given as a relative path, it's base directory is the source directory.
 
 **book.toml**
+
 ```toml
 [book]
 title = "Example book"
 authors = ["John Doe", "Jane Doe"]
 description = "The example book covers examples."
 src = "my-src"  # the source files will be found in `root/my-src` instead of `root/src`
 language = "en"
+logo = "static/logo.png"
```

Comment:
If SVG format is supported, I prefer to use an SVG example.
That way, we don't have to add a sample binary to the Git index, and save space, in addition to the general advantages of SVG logos (crispness, etc.).

Reply:
A quick test showed that SVG seems to be supported. Is there a logo I could use as a sample? If not, I'd just reuse the favicon.

Reply:
I think MdBook doesn't have a product logo yet. Using the favicon as sample seems fine, though .ico files may not be rendered correctly as source image for `img`-elements sometimes.